File: challenge3/birthdayCakeCandles.py
def birthdayCakeCandles(candles):
    # Write your code here
    orderedCandles = sorted(candles, reverse=True)
    max = orderedCandles[0]
    maxFilterCandles = list(filter(lambda x: x == max, orderedCandles))
    return len(maxFilterCandles)

if __name__ == "__main__":
    birthdayCakeCandles([3, 2, 1, 3]); # 2

File: challenge3/birthdayCakeCandles.js
function birthdayCakeCandles(candles) {
    // Write your code here
    const orderedCandles = candles.toSorted((a, b) => b - a)
    const max = orderedCandles[0]
    const maxFilterCandles = orderedCandles.filter(candle => candle === max)
    return maxFilterCandles.length
}

birthdayCakeCandles([3, 2, 1, 3]); // 2